ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+
```

Comment:
Rule of thumb: if your issue doesn't (potentially) require changes to the code-base to be solved, then StackOverflow is probably a better place to bring it up.